operators/db2/secret/ibm-entitled-registry-credentials-secret.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: ibm-entitled-registry-credentials
+  namespace: db2
+spec:
+  encryptedData:
+    IBM_ENTITLED_REGISTRY_PASSWORD: AgAG13RCxVuk7TCjaCQPQlQg8dOhsn6IZUH/YmnNpyPf6Z7A0C+NbuiNC9ciqfTMeQZJ6leIZc3/uJcoRv/ucIEKdQXXY3Rlfk7GSVr46hw/+94xzSrgh7IJxSUBQr7oKVtYuo5tIpFFcDfCI7YRn/ZHaP72BzXq2L6tTHE/PATefE6vg2GDUw4cm3v92+v995VTvWlGtQIndM2ni2Bb81F9xW1K376IsSD8KET1UMuROxaPwgRuFOLPClqBnNnQryPrRVyrPn0hVt9tjq4xjKX59zjb9Grm81ZBzTbwatrDzQOKc/dPwNqyhb07Pq4tj78XN8zROT5cRgGwgfu8FmooTZAPXQ0dKWSv0SYgtT5jfZvtjiwppnGupdT5IgzjiQXxcWtWvPp/0Dtxf6qDJbZZ/8tqz1HiJ4xkp0kW5CWEgBD1hyGuuPZI4TjUECuQuEEMGrjoYc4CES668kIh6e92rPI/svlcVKCy3g/QTzjDlqHKCpNa3hJADh6986SDl/SSYW2VNAkwT+jGg+iIHfB1JQKrNzyhB8Sf0TN4bmSorIvdVwkKsuGFQWNZ9KJM4JteGNvpfgdqdlEo0WNVHWWo9u1fCtT2cUr+TpFTLG2W3bLchzB1X9CKLPw17q7g72hMOEYVRVtnQglCICOU6b4M2w5x9FWFFTi08jkXxW3GcGus+Ms5/P30ideoxXdIiZZowp5U1GUF3ZjcSH37l3dCaIA6e+xaKAdBtnPCg4pCgh0chzgwQoabnkZuMl37ocITL9xM3exIetmYt6Ee6a5dBdKePP/2VNgiFozQw8hvD97q61dqIU7I04ppoKReumbW+NBIJb/43uqJf/o2sDxJp4HE+7xAZVwtu1ZgOSPiTcCoGDppHlplrBWP6yeujBLnSp1STA6ktPUKHkcxipmbfYIQmugS58fATcu3xKvbjA1aVqg57zkD9sAOTJXtmDI=
+    IBM_ENTITLED_REGISTRY_USER: AgB5oN0CCKcOvqfbcTICfv6OmKziB77LM1wHz9S9kdKtC58Fpc+ybz5mburlOQYjMfaKYgrzQyDNt7zk20P7Tr/CbVAdgGh93QSVcg1WKwgvOVG5vTIHiB+9uHENzsqCSrbFjv2xDK9JbrhCRjHd46yHo1abfcCFb4bTJwgAdYi/1fYfxO2XdVkfsXC+NyC4dZWOnk+3kqrXuQS/3tlQ3exY9Y+k3o8cpqvYPi0DIkej6jfHsUJtXchPaEG7t2ao02Nm0Nke0mea2yEvMNIjVAJezmWxruhriuPHnuAzmABm+glIy6utVrE0cwGyuZeEyo3jyUlVafxwWd5lnbcogFZr0y0O3PdbasmyWlO0zqsDcxN2rrXTSQs4FUZG3IXxBSx6FlecvsVs2XW7sgcdTo7LX+VG6tcvR7sqrwWCHouCtTZXP1YciRfMyRuIqBzvi/X0QQRob945MBJLmbak98SNWH51Q5e/+qkjKAoDnO3R67RRhqtvkqRLlIYgMh/UXdzTdZcT+gPB0AJ1jDjy1Na2ytRFAKO4sEwergng0dIVMp/n+vYhE4RSASOtDhrJCCf2eDRg2LLsCMURzgj7tELij15s6u0bjdNRBE2ZOQzj5DzxQQQWAMyfNBYL3AEwnPlLZ7gyHKGwk9MupzAJU0VteGBMWC85v26A+tfa/BCjB2oQwoCihd25o6Y9JmhfBPeISw==
+  template:
+    data: null
+    metadata:
+      creationTimestamp: null
+      name: ibm-entitled-registry-credentials
+      namespace: db2
```

Comment:
Can you also add a "type" field and set it to kubernetes.io/.basic-auth as opposed to kubernetes.io/dockerconfigjson as given in the example below. Sorry about the crappy indendentation, the type field should be in the same indentation as spec.template.metadata stanza in your case

apiVersion: bitnami.com/v1alpha1
kind: SealedSecret
metadata:
  name: ibm-entitlement-key
  namespace: db2
spec:
  encryptedData:
    .dockerconfigjson: XXX
  template:
    metadata:
      name: ibm-entitlement-key
      namespace: db2
    type: kubernetes.io/dockerconfigjson